content/c-sharp/concepts/enums/enums.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'Enums'
+Description: 'An enum (or "Enumeration type") is a set of named labels that each represent a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Enums'
+  - 'Data Types'
+  - 'Arrays'
+  - 'Vectors'
+  - 'Pointers'
+  - 'Memory'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+## Syntax
+
+By default, each entry in an enum is assign a zero-indexed value.
+
+```cs
+enum NinjaTurtle {
+  Leonardo, // 0
+  Michelangelo, // 1
+  Donatello, // 2
+  Raphael // 3
+}
+
+Console.WriteLine((int)NinjaTurtle.Raphael == 3); // prints "True"
+```
+
+However, you can also choose to assign specific number values.
+
+```cs
+enum BreadCount {
+  Dozen = 12,
+  BakersDozen = 13,
+  Gross = 144,
+  GreatGross = 1728
+}
+```
+
```

Comment:
I'd like to see an additional example. (Not a codebyte, just an additional example beforehand.)